pkg/resourcemanager/webhook/projectedtokenmount/handler.go

```diff
@@ -50,12 +53,16 @@ func (h *handler) InjectDecoder(d *admission.Decoder) error {
 }
 
 func (h *handler) Handle(ctx context.Context, req admission.Request) admission.Response {
+	if req.Operation != admissionv1.Create {
```

Comment:
What if a pod gets updated with an additional container? Then our volume mount will not be added to it, or? Is this fine?

Reply:
Such updates are not allowed, see https://kubernetes.io/docs/concepts/workloads/pods/#pod-update-and-replacement:

> Pod updates may not change fields other than `spec.containers[*].image`, `spec.initContainers[*].image`, `spec.activeDeadlineSeconds` or `spec.tolerations`. For `spec.tolerations`, you can only add new entries.